### ISSUE TITLE
CanvasControl & CanvasVirtualControl does not disable the accessibility view.

### ISSUE THREAD

**Issue by coder89** [Microsoft/Win2D#491]
Both canvas controls contain <Image /> XAML control. And that inner control does not have accessibility view set to "Raw" which hides it from the Narrator. Because of that if user enables the Narrator and starts navigating with "Caps + Left/Right arrow key" user will hear "Image" and because that image is hidden under the canvas control implementation, apps would have to try to workaround this issue like we did in our app (on page load we execute this code):

`auto image = m_win2dCanvas->Content;`
`if (image != nullptr)`
`{`
`    image->IsDoubleTapEnabled = false;`
`    image->IsHitTestVisible = false;`
`    image->IsHoldingEnabled = false;`
`    image->IsRightTapEnabled = false;`
`    image->IsTapEnabled = false;`
`    image->SetValue(Control::IsTabStopProperty, false);`
`    image->SetValue(AutomationProperties::AccessibilityViewProperty, AccessibilityView::Raw);`
`}`

This should prevent the inner Image control from being focused, tapped, clicked and navigated in the Narrator mode. This is really important for Microsoft that is pushing for apps being more accessible for people.

**Comment by shawnhar** [comment]
This is fixed in Win2D 1.21.0 (released today)
